src/components/__test__/ReduxStore.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from '../redux/store';
+import App from '../../App';
+
+describe('App component', () => {
+  test('renders', () => {
+    const element = render(
+      <Provider store={store}>
+        <BrowserRouter>
+          <App />
+        </BrowserRouter>
+      </Provider>,
+    );
+    expect(element).toBeTruthy();
+  });
```

Comment:
Since the previous reviewer did not point this out, this comment will be marked as optional in this PR.
[OPTIONAL] Following the project requirements, you should use also React Testing Library to simulate user interaction. Check out this [link ](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/articles/testing_react_with_react_testing_library.md)to learn how to do that.

Reply:
Okay, thanks